test/src/audits/manifest/short-name-length.js

```diff
@@ -27,15 +27,29 @@ describe('Manifest: short_name_length audit', () => {
 
   // Need to disable camelcase check for dealing with short_name.
   /* eslint-disable camelcase */
-  it('fails when a manifest contains no short_name', () => {
+  it('fails when a manifest contains no short_name and no name', () => {
+    const inputs = JSON.stringify({
+      name: null,
+      short_name: null
```

Comment:
should live in the `short-name.js` test since this isn't really testing name length